modules/farm_fd2_school/src/entrypoints/FD1/App.vue

```diff
@@ -176,6 +180,27 @@ export default {
       this.unit = null;
       this.comment = '';
     },
+    async fetchCrops() {
+      const cropsResponse = await fetch(
+        'http://farmos/api/taxonomy_term/plant_type'
+      );
+      const crops = await cropsResponse.json();
+      this.cropList = crops.data;
+    },
+    async fetchPlants() {
+      const plantsResponse = await fetch(
+        'http://farmos/api/fd2_plant_assets?crop=' + this.crop.attributes.name
+      );
+      const plants = await plantsResponse.json();
+      if (Array.isArray(plants)) {
+        this.plantList = plants;
+      } else {
+        this.plantList = [];
+      }
+    },
+  },
+  async created() {
+    this.fetchCrops();
```

Comment:
- The `fetchCrops` method is `async`.  So when it is called using `await` in your `created` lifecycle hook